themes/default/content/docs/get-started/_index.md

```diff
@@ -52,3 +61,18 @@ Select one of the following options to learn how to deploy cloud resources using
         <p>Deploy your first Pulumi program in a live, <a href="{{< relref "/resources/introduction-to-pulumi" >}}">instructor-led event</a>.</p>
     </div>
 </div>
+
+## Additional resources
+
+The following sections are also useful when first learning how to use Pulumi:
+
+<div class="md:flex flex-row mt-6 mb-6">
+    <div class="md:w-1/2 border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts" >}}"><i class="fas fa-file-alt pr-2"></i>Architecture & Concepts</a></h3>
+        <p>Get details on the Pulumi programming model and core concepts.</p>
+    </div>
+    <div class="md:w-1/2 md:ml-4 border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/guides/adopting" >}}"><i class="fas fa-cloud pr-2"></i>Adopting Pulumi</a></h3>
+        <p>Learn how to support, migrate, or convert existing cloud infrastructure with Pulumi.</p>
+    </div>
+</div>
```

Comment:
I do think Tutorials are a particularly valuable thing to link here - as (1) applying Pulumi to a specific problem is one of the most common starting points for users new to Pulumi and (2) they are not particularly prominently linked anywhere else except the end of the Getting Started guides.

Reply:
I agree that they're valuable; however, I don't believe that we should have them be the first place a user goes as part of their initial learning experience. IMHO, they're a little too deep to show someone at the beginning.

We can do a better job of cross-linking, but we have the section link in the left nav and will have one in the top nav under "Learn".